examples/hello/hdf5SubFile/hdf5SubFile.cpp

```diff
@@ -14,7 +14,9 @@
 #include <iostream> //std::cout
 #include <mpi.h>
 #include <stdexcept> //std::invalid_argument std::exception
+#ifndef _MSC_VER
 #include <unistd.h>
+#endif
```

Comment:
I was wondering why the windows test was failing for Liz's PR. I wonder how this was not an issue before

Reply:
Not sure, but fortunately not too difficult a fix.  Probably could have just killed the unistd include, but there were further warnings because visual studio is anal about type conversion warnings.

Reply:
I rebased Liz's PR over this one and now it passes the windows CIs :)